setup.py

```diff
@@ -371,23 +371,22 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'dumb-init>=1.2.2',
     'flake8>=3.6.0',
     'flake8-colors',
+    'flaky~=3.6.1',
     'freezegun',
     'ipdb',
     'jira',
     'mongomock',
     'moto==1.3.5',
-    'nose',
-    'nose-ignore-docstring==0.2',
-    'nose-timer',
     'parameterized',
     'paramiko',
     'pre-commit',
     'pylint~=2.3.1',  # to be upgraded after fixing https://github.com/PyCQA/pylint/issues/3123
                       # We should also disable checking docstring at the module level
     'pysftp',
+    'pytest',
+    'pytest-cov==2.8.1',
```

Comment:
Why do you think it makes sense to pin it?

Personally I never had issues with pytest-cov.